solutions/usaco-762.mdx

```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
 
 **Time Complexity:** $\mathcal{O}(N)$
 
-<LanguageSection>
+<LanguageSection> 
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+typedef long long ll;
+
+int main() {
+        freopen("homework.in", "r", stdin);
+        ll n;
+        cin >> n;
+        vector<ll> questions(n);
+        for (ll i = 0; i < n; i++) {
+                cin >> questions[i];
+        }
+        ll left = -1;
+        ll right = n;
+
+        vector<ll> total(n, 0);
+        vector<ll> minimum(n, 0);
+        vector<ll> max_score_values;
+
+        total[n - 1] = questions[n - 1];
+        minimum[n - 1] = questions[n - 1];
+
+        ll cur_q, cur_s, idx;
```

Comment:
expand these variable names

Reply:
unaddressed

Reply:
I have made the changes, but the deployment was failed sorry, I have again committed the changes.

Reply:
don't think it's addressed still?